### DISCUSSION TITLE
Airplay not working when installed with Synology docker

### DISCUSSION THREAD

**Discussion by hanbo1990** [music-assistant/support#1335] [Q&A]
airplay reports: 
RuntimeError: Unable to locate RaopBridge for Linux (x86_64)

Software version: Latest music assistant server docker
Synology 920+ with DSM 7.1.1-42962 Update 5

**Comment by marcelveldt** [comment]
Try updating or reinstalling the Synology OS, we've seen this issue being reported in the past and in all cases that fixed it.
